lib/actions/lookupFromDictionary.js

```diff
@@ -0,0 +1,53 @@
+const parser = require('papaparse');
+const { messages } = require('elasticio-node');
+
+exports.process = async function process(msg, cfg) {
+  if (cfg.from === cfg.to) throw new Error('Please select different columns to translate between');
+
+  const parsedData = parser.parse(cfg.table, { header: true });
+  const row = parsedData.data.filter((r) => r[cfg.from] === msg.body.input);
+
+  if (!row.length && cfg.emitEmptyObject) {
+    await this.emit('data', messages.newEmptyMessage());
+    return;
+  }
+  if (!row.length) throw new Error('Value does not exist in dictionary selected');
+  if (row.length !== 1) throw new Error('Error in locating value');
+
+  await this.emit('data', messages.newMessageWithBody({ result: row[0][cfg.to] }));
```

Comment:
You can use`return` for one result without check, instead of calling `emit`

Reply:
Is that preferred to calling emit? Why?

Reply:
In this case it's the same, but return is more readable, and node js won't wait for await emit inside process method

Reply:
Done!